model_compression_toolkit/qat/keras/quantizer/quantization_builder.py

```diff
@@ -50,10 +50,10 @@ def quantization_builder(n: common.BaseNode,
     if n.is_weights_quantization_enabled():
         quant_method = n.final_weights_quantization_cfg.weights_quantization_method
 
-        quantizer_class = get_quantizer_class(QuantizationTarget.Weights,
-                                              qat_config.weight_training_method,
-                                              quant_method,
-                                              BaseKerasQATTrainableQuantizer)
+        quantizer_class = get_trainable_quantizer_class(QuantizationTarget.Weights,
+                                                        qat_config.weight_training_method,
+                                                        quant_method,
+                                                        BaseKerasQATTrainableQuantizer)
         attributes = fw_info.get_kernel_op_attributes(n.type)
```

Comment:
instead of renaming the function, I guess we could use this argument 'BaseKerasQATTrainableQuantizer'.
I'm not sure it's dramatic... just a comment.

Reply:
I prefer no to use the class type (or in that case even the type's name) to deduce the type of filtering that should be applied (this is the main difference between the "get_trainable" and "get_inferable" functions).
If you think that this is important then let's discuss it, if not then I prefer to keep the current implementation.

Reply:
I tend to agree.